duckdb-vortex/src/include/vortex_common.hpp

```diff
@@ -107,12 +108,21 @@ struct Array {
 	vx_array *array;
 };
 
+
 struct ArrayIterator {
 	explicit ArrayIterator(vx_array_iterator *array_iter) : array_iter(array_iter) {
 	}
 
+	vx_array_iterator* release() {
```

Comment:
Can we add some docs here that explain how memory ownership is handled using release?